### ISSUE TITLE
help prints out static default loss function

### ISSUE THREAD

**Issue by avanwouwe** [ivanhk/fastText_java#27]
in line 90 of Args we print a default loss function of "ns". However this is not true for supervised mode, where the default is softmax. It should use the current value of the arg, like all the other parameters:

`				+ "  -loss               loss function {ns, hs, softmax} [" + loss + "]\n"`
